pkg-sync/README.md

```diff
@@ -2,11 +2,34 @@
 
 A tool for syncing packages in an on-prem builder instance with the packages in public builder.
 
-Given an origin (defaults to `core`), channel (defaults to `stable`) and an on-prem builder url and token, this tool can download all packages in the origin and channel from the public Chef Habitat builder that the on-prem instance does not already have and upload them to the on-prem builder.
+## What is Package Syncing?
+
+Package syncing allows you to keep your on-premises Habitat Builder instance up-to-date with packages from the public/SaaS Chef Habitat Builder. This is essential for:
+- Maintaining access to the latest core packages and dependencies
+- Ensuring your on-prem builder has all necessary packages for building and running applications
+- Synchronizing specific package sets like Habitat CLI tools or Builder components
+
+For **Habitat 2.0**, the default channel has changed from `stable` to `base` to provide better lifecycle management and compatibility.
+
+Given an origin (defaults to `core`), channel (defaults to `base`) and an on-prem builder url and token, this tool can download all packages in the origin and channel from the public Chef Habitat builder that the on-prem instance does not already have and upload them to the on-prem builder.
 
 This performs a pre-flight check to ensure that you do not have local packages in the channel that are not in the same channel on bldr.habitat.sh. If there are, these local packages must be demoted.
 
-Optionally, one can specify a `--package-list` with a value of either `habitat` or `builder`. Rather than syncing all of the latest packages in the origin/channel, this will sync a predefined list of pckages. Specifying `habitat` will sync all packages included in a habitat release (cli, supervisor, studio, etc.). Specifying `builder` will sync all packages needed to run an on-prem builder instance. Note that when providing a `--package-list`, the above pre-flight check is not performed.
+## Package List Options
+
+Optionally, one can specify a `--package-list` with a value of either `habitat` or `builder`. Rather than syncing all of the latest packages in the origin/channel, this will sync a predefined list of packages:
+
+- **`habitat`**: Syncs all packages included in a habitat release (CLI, supervisor, studio, etc.)
+- **`builder`**: Syncs all packages needed to run an on-prem builder instance
+
+Note that when providing a `--package-list`, the above pre-flight check is not performed.
+
+## Native Packages
```

Comment:
I would not add this section. Customers should not be working directly with native packages in their own workflows.